docs/user/icl/general.rst

```diff
@@ -0,0 +1,12 @@
+.. _icl-general-information:
+
+General Information
+===================
+
+Artemis offers the Integrated Code Lifecycle (ICL), a comprehensive feature designed to streamline the development process for programming exercises.
+ICL combines version control, secure communication, and continuous integration to provide a seamless experience for students and instructors.
+Understanding these components is crucial for effectively using Artemis, especially if you're new to concepts like SSH and Git. The key components of ICL are:
```

Comment:
We still support gitlab (for how long?) but this sentence sounds like there is no alternative to ICL. Especially if the instance uses gitlab instead of ICL, this might confuse instructors

Reply:
That's true, but gitlab will be discontinued rather soon, so I think it's ok